src/main/java/com/univocity/parsers/common/processor/RowListProcessor.java

```diff
@@ -90,4 +92,18 @@ public void processEnded(ParsingContext context) {
 	public String[] getHeaders() {
 		return headers;
 	}
+
+    /**
+     * @return list of {@link Row} objects, one for each row of the csv file.
+     */
+  	public List<Row> getRowObjects() {
+	  Csv csv = new Csv(headers);
+	  List<Row> rowList = new ArrayList<Row>();
+	  int i = 1;
```

Comment:
You'll also get a line number from the `ParsingContext` object that is sent by the parser to the `RowProcessor.rowProcessed` method - no need to count again.